tests/theme/oe_bootstrap_theme_test_subtheme/oe_bootstrap_theme_test_subtheme.info.yml

```diff
@@ -0,0 +1,8 @@
+name: OpenEuropa Bootstrap Theme Test Subtheme
```

Comment:
Didn't we say about using the already existing test themes?

Reply:
yep